server/backend/src/users/users.controller.ts

```diff
@@ -41,4 +43,20 @@ export class UsersController {
       }
     }
   }
+
+  @Put(':id/notifyToken')
+  async updateNotifyToken(
```

Comment:
maybe rename to something like `updateNotifyTokenEndpoint` since theres another method called updateNotifyToken but ehhhhhhhhhhhhhhhhhhhhhhhhhhhh up to you :)

Reply:
I'm not gonna change it cus I'm stupid (also it's inside a controller so I think that it being an endpoint is implied??? idk)